integration/helpers/ports.go

```diff
@@ -38,168 +41,207 @@ func init() {
 	}
 }
 
+// NewPortValue is deprecated. Do not use in new code. Use NewListener() and
+// friends instead.
```

Comment:
Total nit-pick, [but if you use `Deprecated: ` then it'll show as deprecated in various editors and godoc.](https://rakyll.org/deprecated/)

Reply:
Done